lib/extensions.js

```diff
@@ -77,6 +77,28 @@ module.exports = function (realmConstructor) {
     return `${name}#${id}`;
   };
 
+  // Overriding Object.is()
+  Object.is = function (x, y) {
```

Comment:
This should be moved to a separate file that is manually required in by the user, to ensure it's an opt-in feature.